Dockerfile

```diff
@@ -8,7 +8,6 @@ RUN apt-get update \
         apt-transport-https \
         ca-certificates \
         cabextract \
-        git \
```

Comment:
I know it’s extra bloat, but I added `git` on request for another user so could you please not remove it?